<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 208, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRegistrarcli">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>350</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>registrar nuevo cliente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegistrartrab">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>350</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Century Gothic&quot;;
</string>
    </property>
    <property name="text">
     <string>registrar nuevo trabajo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscarcli">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>350</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>buscar clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGanancia">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>350</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>ganancias</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInsumos">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>350</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>insumos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Software para consultor en TI</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/loogoo/WhatsApp Image 2021-11-15 at 19.03.07.jpeg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
